api/src/main/resources/workbench.yaml

```diff
@@ -1100,6 +1100,22 @@ paths:
           schema:
             $ref: "#/definitions/Cohort"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/{cohortId}/duplicate:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+      - $ref: '#/parameters/cohortId'
+    post:
+      tags:
+        - cohorts
+      description: Creates a duplicate of the cohort specified by cohortId
+      operationId: "duplicateCohort"
+      responses:
+        200:
+          description: The cohort definition that was created.
+          schema:
+            $ref: "#/definitions/Cohort"
```

Comment:
Nit: This can also return a BadRequestException, right? If so we should document.